src/components/comments/Answer.js

```diff
@@ -1,14 +1,21 @@
 import React from "react";
 import "../mainContent/mainContent.css";
+import "./comments.css";
 export default function Answer(props) {
   return (
-    <div className="answer-bubble comment-bubble ">
-      {/* Puts the answer header and answer together within the same bubble
-      I think having either only answerText or answerHeaderText for Answer components 
-      can make it so that our answers can have just bold text or just unbolded text, what 
-      issues can we run into down the line using this approach?*/}
-      <div className="body-text text-bold">{props.answerHeaderText}</div>
-      <div className="body-text">{props.answerText}</div>
+    <div className="single-comment-container answer">
+      <img
+        className="comment-profile-pic"
+        src={props.pic.profilePic}
+        alt="An icon that matches up with the post"
+      />
+      <div className="comment-bubble ">
+        <div className="body-text text-bold">{props.answerHeaderText}</div>
+        <div className="body-text" dangerouslySetInnerHTML={{ __html: props.answerText}}></div>
+        {/* dangerouslySetInnerHTML allows json strings to be interpreted as html. 
+        This is done so that certain segments of the string can be bolded. 
+        dangerouslySetInnerHTML cannot be used for posts because React components for interactive portions cannot be interpreted properly.*/}
+      </div>
```

Comment:
`props.answerText` is a json string. It's inside `labContent.js` -> `followups` -> `answer`. If I pass in the string without dangerouslySetInnerHTML then the HTML tags inside the string aren't rendered properly :/

Reply:
Hm, so instead of making the type of the `answer` a JSON string, you can consider making it a JSX element (e.g. an enclosing HTML tag + normal markup inside). Then, you should be able to render the `answer` as an element, which applies HTML/CSS as needed, without using `dangerouslySetInnerHTML`. Let me know if that doesn't work!